source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/show/header.tsx

```diff
@@ -2,32 +2,30 @@ import * as React from 'react';
 
 import { ProjectResource, attributesFor } from '@data';
 
-import { i18nProps, withTranslations } from '@lib/i18n';
+import { useTranslations } from '@lib/i18n';
 import TimezoneLabel from '@ui/components/labels/timezone';
 
-type IProps = { project: ProjectResource } & i18nProps;
+interface IProps {
+  project: ProjectResource;
+}
 
-class Header extends React.PureComponent<IProps> {
-  render() {
-    const { project, t } = this.props;
-    const { name, dateCreated, language } = attributesFor(project);
+export default function Header({ project }: IProps) {
```

Comment:
converted to a functional component to try to track down a state mutation after render